README.md

```diff
@@ -155,9 +172,25 @@ PS> scoop bucket add extras
 PS> scoop install go-ipfs
 ```
 
+### Install prebuilt packages
+
+[![dist.ipfs.io Downloads](https://img.shields.io/github/v/release/ipfs/go-ipfs?label=dist.ipfs.io&logo=ipfs&style=for-the-badge&cacheSeconds=3600)](https://ipfs.io/ipns/dist.ipfs.io#go-ipfs)
+
+From there:
+- Click the blue "Download go-ipfs" on the right side of the page.
+- Open/extract the archive.
+- Move `ipfs` to your path (`install.sh` can do it for you).
+
+You can also download go-ipfs from this project's GitHub releases page if you are unable to access dist.ipfs.io:
+
+[![dist.ipfs.io Downloads](https://img.shields.io/github/v/release/ipfs/go-ipfs?label=GitHub%20Downloads&logo=ipfs&style=for-the-badge&cacheSeconds=3600)](https://github.com/ipfs/go-ipfs/releases)
+
+
 ### Build from Source
 
-go-ipfs's build system requires Go 1.15.2 and some standard POSIX build tools:
+![GitHub go.mod Go version](https://img.shields.io/github/go-mod/go-version/ipfs/go-ipfs?label=Requires%20Go&logo=go&style=flat-square&cacheSeconds=3600)
```

Comment:
Wow. This is GREAT! (we're terrible at updating the readme to reflect the latest version)